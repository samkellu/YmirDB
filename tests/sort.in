set a 123 33 22 34 55656 67 77 12 234 345 75676 234 65647 8689 22 34 55
sort a
set b 1 1 1 1 1 123 33 22
get a
get b
sort b
list entries
sort d
bye
