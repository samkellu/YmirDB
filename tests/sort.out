> ok

> ok

> ok

> [12 22 22 33 34 34 55 67 77 123 234 234 345 8689 55656 65647 75676]

> [1 1 1 1 1 123 33 22]

> ok

> b [1 1 1 1 1 22 33 123]
a [12 22 22 33 34 34 55 67 77 123 234 234 345 8689 55656 65647 75676]

> no such key

> bye
